<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NfsShare</class>
 <widget class="QDialog" name="NfsShare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NfsShare</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Nfs shared</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="add_nfs">
          <property name="text">
           <string>Add NFS shared</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/nfs.png</normaloff>:/images/nfs.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="modify_nfs">
          <property name="text">
           <string>Modify NFS</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/log.png</normaloff>:/images/log.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_nfs">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTextEdit" name="textEditDebug">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="CheckBox" name="checkBoxTerminal"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>View terminal service</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start service</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/service.png</normaloff>:/images/service.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRestart">
       <property name="text">
        <string>Restart service</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/service.png</normaloff>:/images/service.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop service</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/service.png</normaloff>:/images/service.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStatus">
       <property name="text">
        <string>Status service</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/service.png</normaloff>:/images/service.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Apply modified</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QWidget</extends>
   <header location="global">CondresControlCenter/CheckBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>add_nfs</tabstop>
  <tabstop>modify_nfs</tabstop>
  <tabstop>remove_nfs</tabstop>
  <tabstop>pushButtonStart</tabstop>
  <tabstop>pushButtonRestart</tabstop>
  <tabstop>pushButtonStop</tabstop>
  <tabstop>pushButtonStatus</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
